test/plugins/validation/2and3/info.js

```diff
@@ -0,0 +1,98 @@
+const config = require('../../../../src/.defaultsForValidator').defaults.shared;
+const expect = require('expect');
+const {
+  validate
+} = require('../../../../src/plugins/validation/2and3/semantic-validators/info');
+
+describe('validation plugin - semantic - error handling for incorrect formatting', () => {
```

Comment:
Have this describe message be `validation plugin - semantic - info`. I'm pretty sure we usually use the name of the validator (so the name of the file) for that last bit